RunJavaUtils.py

```diff
@@ -128,23 +129,21 @@ def Run_Java_File(destName, package, baseName):
         fullClassName = os.path.join(package, className)
         invokeName = package + "." + invokeName            
     try:
-        subprocess.check_output(["javac", destName], stderr=True)
-    except subprocess.CalledProcessError:
-        output = ["build error"]
+        subprocess.check_output(["javac", destName])
+    except subprocess.CalledProcessError as e:
+        output = ["build error: " + str(e.output)]
         success = False
     if (success == True):
         try:
             if (package != ""):
                 move(className, fullClassName)
-            binaryOutput = subprocess.check_output(["java", invokeName], stderr=True)
-            output=Change_Binary_To_String_List(binaryOutput)                        
-        except subprocess.CalledProcessError:
-            output = ["run error"]
+            raw_output = subprocess.check_output(["java", invokeName], stderr=subprocess.STDOUT)
+            output = raw_output.splitlines()
+        except subprocess.CalledProcessError as e:
+            output = ["run error: " + str(e.output)]
             success = False
-        try:
+        if os.path.exists(fullClassName):
             os.remove(fullClassName)
-        except FileNotFoundError:
```

Comment:
Are we expecting to be writing all this stuff to a temporary directory? If so, we could omit removing files and just rely on one top-level `shutil.rmtree` at the end.

Reply:
I added this exception when trying to pull this file
https://classroom.google.com/u/0/g/tg/MTQ5MTE3MDA5NDFa/MTY2ODA4NjEzMTda#NDA1MjAyODla
from the google drive.  It won't pull down because he created a file with intentional corrupt strings.  Probably a cleaner way to handle it, but this let me continue downloading the rest of the docs.